neo-cli/Settings.cs

```diff
@@ -66,15 +66,17 @@ internal class RPCSettings
         public ushort Port { get; }
         public string SslCert { get; }
         public string SslCertPassword { get; }
-        public Fixed8 MaxGasInvoke { get; }
+        public Fixed8 ExtraGasInvoke { get; }
+        public int MaxConcurrentConnections { get; }
 
         public RPCSettings(IConfigurationSection section)
         {
             this.BindAddress = IPAddress.Parse(section.GetSection("BindAddress").Value);
             this.Port = ushort.Parse(section.GetSection("Port").Value);
             this.SslCert = section.GetSection("SslCert").Value;
             this.SslCertPassword = section.GetSection("SslCertPassword").Value;
-            this.MaxGasInvoke = Fixed8.Parse(section.GetValue("MaxGasInvoke", "0"));
+            this.ExtraGasInvoke = Fixed8.Parse(section.GetValue("ExtraGasInvoke", "0"));
+            this.MaxConcurrentConnections = int.Parse(section.GetValue("MaxConcurrentConnections", "120"));
```

Comment:
`120` is too high for normal use.

```suggestion
            this.MaxConcurrentConnections = int.Parse(section.GetValue("MaxConcurrentConnections", "10"));
```

Reply:
> MaxConcurrentConnections

`10` concurrency is too low for a full node rpc. Mainnet seed nodes is meeting more than `40` concurrency now. And rising it to this amount doesn't influence node stability or performance.

Reply:
You can change the configuration file for seed nodes. But the default settings of the configuration files should suit the needs of normal users.

Reply:
> You can change the configuration file for seed nodes. But the default settings of the configuration files should suit the needs of normal users.

Fine.